### DISCUSSION TITLE
fastlane feature to expire particular build in TestFlight

### DISCUSSION THREAD

**Discussion by nawaahme** [fastlane/fastlane#18211] [Ideas]
Can we have a fastlane feature to expire particular build in TestFlight? 

Currently, fastlane only supports parameter "expire_previous_builds"  which expires only previous builds if passed as an argument.

**Comment by nickfourtimes** [comment]
This would be especially useful since it seems that we've occasionally a limit on numbers of builds currently on TestFlight (it might have been macOS-specific?). We might want to expire only the oldest, say, 5-10 builds in that case, but the `expire_previous_builds` [parameter](http://docs.fastlane.tools/actions/testflight/#parameters) will completely wipe out _all_ previous builds, which is not what we want.
